### ISSUE TITLE
[kubeflow v1.3.0-rc.1] cache server certificate secret not being created

### ISSUE THREAD

**Issue by raffaelespazzoli** [kubeflow/manifests#1853]
I'm trying installing kubeflow v1.3.0-rc.1 with the official manifests and it seems to me that the manifests for creating the certs for the cache-server are not present.
It is still hard to say for creating for sure as the kustomize composition is still very hard to follow. That said, thanks for having removed kfctl and kfdef.

**Comment by Bobgy** [comment]
Hi @raffaelespazzoli, can you report your kubernetes version?
